sdk/storage/storage-blob-changefeed/review/storage-blob-changefeed.api.md

```diff
@@ -0,0 +1,85 @@
+## API Report File for "@azure/storage-blob-changefeed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BlobServiceClient } from '@azure/storage-blob';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+
+// @public (undocumented)
+export class BlobChangeFeedClient {
+    constructor(blobServiceClient: BlobServiceClient);
+    // (undocumented)
+    getChanges(options?: BlobChangeFeedGetChangesOptions): PagedAsyncIterableIterator<BlobChangeFeedEvent, BlobChangeFeedEventPage>;
```

Comment:
I think paged APIs are supposed to start with `list`

Reply:
Oh, do we have a guideline doc for this kind of thing?

Reply:
https://azure.github.io/azure-sdk/typescript_design.html#ts-client-naming-conventions